integration/spark/spark3/src/test/java/io/openlineage/spark3/agent/lifecycle/plan/catalog/IcebergHandlerTest.java

```diff
@@ -168,6 +169,52 @@ void testGetDatasetIdentifierForRest() {
         .hasFieldOrPropertyWithValue("type", DatasetIdentifier.SymlinkType.TABLE);
   }
 
+  @ParameterizedTest
+  @CsvSource({
+    "jdbc:mysql://test.1234567890.us-west-2.rds.amazonaws.com:3306/database,mysql://test.1234567890.us-west-2.rds.amazonaws.com:3306",
+    "jdbc:postgresql://localhost:5432/mydb?ssl=true&applicationName=myApp,postgres://localhost:5432",
+    "jdbc:oracle:thin:@//myhost:1521/myservice?TNS_ADMIN=/path/to/tns,oracle://myhost:1521",
+    "jdbc:sqlserver://localhost:1433;databaseName=mydb;user=me;password=secret,sqlserver://localhost:1433",
```

Comment:
Wouldnt a single test scenario be enough? Looks like this is testing `JdbcDatasetUtils` here and duplicating tests should be avoided.

Reply:
I disagree - here I care purely about namespace part and stripping of unnecessary params. `JdbcDatasetUtils` uses those to determine database name that's used as part of dataset name.